src/plugins/intel_cpu/src/transformations/tpp/x64/op/eltwise.hpp

```diff
@@ -26,20 +27,25 @@ class EltwiseTPP : public modifier::TensorProcessingPrimitive {
 public:
     static bool is_supported(const std::shared_ptr<ov::Node>& node);
     bool visit_attributes(AttributeVisitor& visitor);
+    virtual OpDescTPP get_op_desc() const  = 0;
 };
 
 class BinaryEltwiseTPP : public EltwiseTPP {
 public:
     BinaryEltwiseTPP(libxsmm_meltw_binary_type op_type);
-    libxsmm_meltw_binary_type get_op_type() const { return m_op_type; }
-private:
+    OpDescTPP get_op_desc() const override { return OpDescTPP(m_op_type, m_flags); }
+    static libxsmm_bitfield get_broadcasting_flags(const ov::PartialShape& pshape_0, const ov::PartialShape& pshape_1);
+    static libxsmm_bitfield get_broadcasting_flags(const snippets::VectorDims& pshape_0, const snippets::VectorDims& pshape_1);
```

Comment:
If I didn't miss anything, these methods can be moved to protected section. If it's so, could you please move them to there?

Reply:
I thought they can be used in the emitters as well, but since it's not the case yet let's move them to protected.